thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -69,11 +69,11 @@ if (!(Test-Path "nunit.framework.dll"))
 
 if (!(Test-Path "windows/SDL2.dll"))
 {
-	echo "Fetching SDL2 from NuGet."
-	./nuget.exe install sdl2 -Version 2.0.3 -ExcludeVersion
-	cp sdl2.redist/build/native/bin/Win32/dynamic/SDL2.dll ./windows/
-	rmdir sdl2 -Recurse
-	rmdir sdl2.redist -Recurse
+	echo "Fetching SDL2 from libsdl.org"
+	$target = Join-Path $pwd.ToString() "SDL2-2.0.4-win32-x86.zip"
```

Comment:
Are you sure this is failing it, because the download works fine and this is just what we do below.